static/app/views/app/index.tsx

```diff
@@ -212,12 +213,14 @@ function App({children, params}: Props) {
 
   return (
     <Profiler id="App" onRender={onRenderCallback}>
-      <MainContainer tabIndex={-1} ref={mainContainerRef}>
-        <GlobalModal onClose={handleModalClose} />
-        <SystemAlerts className="messages-container" />
-        <Indicators className="indicators-container" />
-        <ErrorBoundary>{renderBody()}</ErrorBoundary>
-      </MainContainer>
+      <OrganizationContextProvider>
+        <MainContainer tabIndex={-1} ref={mainContainerRef}>
+          <GlobalModal onClose={handleModalClose} />
+          <SystemAlerts className="messages-container" />
+          <Indicators className="indicators-container" />
+          <ErrorBoundary>{renderBody()}</ErrorBoundary>
+        </MainContainer>
+      </OrganizationContextProvider>
```

Comment:
‼️‼️‼️‼️‼️

This means that we can **now use `useOrganization` inside of modals!**

Reply:
Do we need another ErrorBoundary above the org context provider?

Reply:
I don't think so. App is going to be completely busted if we can't load org either way